releasenotes/notes/move-es-version-vars-17c22cfd558a68d5.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
I was thinking the same thing. Maybe the role author wanted to make it clear that these package versions are being defined statically? Or maybe they wanted to have the ability to override the package version?